UPGRADING.md

```diff
@@ -1,6 +1,16 @@
 Upgrading SlackRubyBot
 ======================
 
+### Upgrading to >= 0.8.0
+
+#### Hook::Base was removed, explicitly register any custom hook classes you have
+
+Hook classes are now handled differently, namely they are explicitly registered into `SlackRubyBot::Server` via a configuration option (`hook_handlers`) or by passing a similar hash later on through the `add_hook_handlers` method. Including Hook classes directly into the server class is no longer needed.
+
+A hook is actioned via a `call` message onto the handler object (this can be anything that responds to that), so you'll need to rename your method names.
```

Comment:
rename your methods ... rename your method names is a bit like "buttered buttered sandwich" in French :)